dev/com.ibm.websphere.appserver.features/visibility/public/netty-1.0/resources/l10n/io.openliberty.netty-1.0.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables support for the Netty transport framework
```

Comment:
I imagine that as/if the UFO resolves that the Netty APIs themselves are not public then this might be a good place to document that.

Reply:
That's a good point, I agree that we'll want to define what `...enables support for the Netty transport framework` actually accomplishes in this description.